tests/console/zypper_lifecycle.pm

```diff
@@ -58,6 +60,13 @@ sub run {
     diag('fate#320597: Introduce \'zypper lifecycle\' to provide information about life cycle of individual products and packages');
 
     select_serial_terminal;
+
+    if (check_var('SLE_PRODUCT', 'sles') && !is_aarch64) {
```

Comment:
Wouldn't be better just to check `SCC_REGCODE_LIVE` exists to avoid product and arch conditions?

Reply:
ok

Reply:
ok, I think it is better with the condition where it's expected and then `get_required_var`, but I don't mind.

Reply:
ok, it will catch wrong openqa setup with required var